streams/src/main/java/org/apache/kafka/streams/kstream/internals/foreignkeyjoin/SubscriptionResponseWrapper.java

```diff
@@ -18,11 +18,10 @@
 
 import org.apache.kafka.common.errors.UnsupportedVersionException;
 
+import java.util.Arrays;
+
 public class SubscriptionResponseWrapper<FV> {
     final public static byte CURRENT_VERSION = 0x00;
-    //Max version is limited by how many bytes we have available in the Serde.
-    //Note: Do not change this such that the value goes below CURRENT_VERSION.
-    final private static byte MAXIMUM_VERSION_INCLUSIVE = (byte) (2 ^ SubscriptionResponseWrapperSerde.versionBits);
```

Comment:
I see what you were doing here, but the field was actually unused. I deleted it as "documentation" of this fact.